docs/outputs.rst

```diff
@@ -366,6 +376,48 @@ For CompCor decompositions, entries include:
     Entries that are not saved in the data file for denoising are still stored in metadata with the
     ``dropped`` prefix.
 
+.. caution::
+    Only a subset of these CompCor decompositions should be used for further denoising.
+    The original Behzadi aCompCor implementation [Behzadi2007]_ can be applied using
+    components from the combined masks, while the more recent Muschelli implementation
+    [Muschelli2014]_ can be applied using
+    the :abbr:`WM (white matter)` and :abbr:`CSF (cerebro-spinal fluid)` masks.
+    To determine the provenance of each component, consult the metadata file (described above).
```

Comment:
It might be worth mentioning the option to include either a fixed number of regressors or sufficient regressors to explain some minimum fraction of variance (e.g., 50%), both of which are possible using the confounds generated by fMRIPrep.

For instance:
```
.. caution::
    There are many valid ways of selecting CompCor components for further denoising.
    In general, the components with the largest singular values (i.e., those that
    explain the largest fraction of variance in the data) should be selected.
    *fMRIPrep* outputs components in descending order of singular value.
    Common approaches include selecting a fixed number of components (e.g., the
    first 5 or 6), using a quantitative or qualitative criterion (e.g., elbow, broken
    stick, or condition number), or using sufficiently many components that a minimum
    cumulative fraction of variance is explained (e.g., 50%)
```